core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -414,6 +417,19 @@ public static boolean verboseResponses() {
         return VERBOSE_RESPONSES;
     }
 
+    /**
+     * Returns the fully qualified class name of {@link RequestContextStorageProvider} that is used to choose
+     * when multiple {@link RequestContextStorageProvider}s exist.
+     *
+     * <p>The default value of this flag is an empty string, which means only one
+     * {@link RequestContextStorageProvider} must be found via Java SPI. If there are more than one,
+     * specify the {@code -Dcom.linecorp.armeria.requestContextStorageProvider=<FQCN>} JVM option to
+     * choose the {@link RequestContextStorageProvider}.
+     */
+    public static String requestContextStorageProvider() {
```

Comment:
I wonder if returning `null` makes more sense when no provider is specified.

Reply:
Done!